.github/workflows/ci.yml

```diff
@@ -9,80 +9,110 @@ on:
       - 'v*'
   pull_request:
 
+env:
+  NODE_VERSION: 16
+
 jobs:
-  test:
+  lint:
+    name: Lint files
     runs-on: ubuntu-latest
-    outputs:
-      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    timeout-minutes: 5
```

Comment:
Good practice to set `timeout-minutes` to a low value so that CI can fail early when something goes wrong. (The default value is `360` minutes.)